tools/dtest/harness/harness.go

```diff
@@ -0,0 +1,305 @@
+package harness
+
+import (
+	// pprof import
+	_ "net/http/pprof"
+
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"sync/atomic"
+	"time"
+
+	"github.com/m3db/m3db/tools/dtest/config"
+	"github.com/m3db/m3db/x/m3em/convert"
+	m3dbnode "github.com/m3db/m3db/x/m3em/node"
```

Comment:
I've got it like this to distinguish imports local to the current repository from third party imports. Any reason to merge?